build.gradle

```diff
@@ -90,6 +96,11 @@ def getClangFormatVersion() {
 
 def awslcSrcPath = "${projectDir}/aws-lc/"
 
+// Check for user inputted AWS-LC source directory. 
+if (System.properties["AWSLC_SRC_DIR"]) {
+    awslcSrcPath = System.properties["AWSLC_SRC_DIR"]
```

Comment:
The error message [here](https://github.com/corretto/amazon-corretto-crypto-provider/blob/main/build.gradle#L177) needs to be updated. In case the user sets this property to a custom path and they have not checked out aws-lc's repo, then running git submodule is not going to help.

Reply:
Updated! Would we have to also add error cases for non-empty directories that don't contain the required source files, or should we let the subsequent build commands throw those errors?

Reply:
I don't see it needed.